spec/core_functions/color/hsl/three_args/units.hrx

```diff
@@ -16,6 +83,38 @@ a {
   b: #bf4040;
 }
 
+<===> saturation/unitless/warning
+WARNING: $saturation: Passing a number without unit % is deprecated.
```

Comment:
Why are some of these deprecation warnings just `WARNING` while others say `DEPRECATION WARNING`?

Reply:
Because I forgot to pass `deprecation: true` to one of my calls to `warn` :stuck_out_tongue_closed_eyes: